src/needs/need.model.ts

```diff
@@ -23,6 +22,11 @@ const NeedSchema = new Schema({
         type: Number,
         require: true
     },
+    fulfilled: {
+        type: Types.ObjectId,
+        ref: "Fulfillment",
+        require: true
```

Comment:
we should not require this, otherwise people wouldnt be able to post a Need without an associated fulfillment. We want users to post the stuff they need, then later someone can fulfill it, which is where this data model field comes into play. Let's remove the `require: true`